rackcity/api/serializers/change_plan_serializers.py

```diff
@@ -0,0 +1,22 @@
+from rest_framework import serializers
+from rackcity.models import ChangePlan
+
+
+class AddChangePlanSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = ChangePlan
+        fields = (
+            'name',
+            'execution_time',
+            'owner'
+        )
+
+
+class GetChangePlanSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = ChangePlan
+        fields = (
+            'name',
+            'execution_time',
+            'id'
+        )
```

Comment:
why are there 2 serializers? they seem identical?

Reply:
added related comment below in views

Reply:
resolved via houseparty

Reply:
haha I am absolutely blind, I didn't notice id and owner were different. carry on